extensions/hibernate-search-elasticsearch/runtime/src/main/java/io/quarkus/hibernate/search/elasticsearch/runtime/HibernateSearchElasticsearchRecorder.java

```diff
@@ -124,6 +124,8 @@ private void contributeBackendRuntimeProperties(BiConsumer<String, Object> prope
                 ElasticsearchBackendRuntimeConfig elasticsearchBackendConfig) {
             addBackendConfig(propertyCollector, backendName, ElasticsearchBackendSettings.HOSTS,
                     elasticsearchBackendConfig.hosts);
+            addBackendConfig(propertyCollector, backendName, ElasticsearchBackendSettings.PROTOCOL,
+                    elasticsearchBackendConfig.protocol.getHibernateSearchString());
```

Comment:
Can you add a note here: https://github.com/quarkusio/quarkus/wiki/Migration-Guide ?

Reply:
As a matter of fact, I cannot. I don't have the sufficient rights.

Here's something, if you want to add it yourself:

````
### Hibernate Search + Elasticsearch (Preview)

The protocol used to contact Elasticsearch is now specified in a separate configuration property, `quarkus.hibernate-search.elasticsearch.protocol`. This property defaults to `http`. 

As a result, the property `quarkus.hibernate-search.elasticsearch.discovery.default_scheme` was removed.

So if your configuration looked like this:

```
quarkus.hibernate-search.elasticsearch.hosts = http://es1.mycompany.com
```

You should now have this:

```
quarkus.hibernate-search.elasticsearch.hosts = es1.mycompany.com
```

And if it looked like this:

```
quarkus.hibernate-search.elasticsearch.hosts = https://es1.mycompany.com
quarkus.hibernate-search.elasticsearch.discovery.enabled = true
quarkus.hibernate-search.elasticsearch.discovery.default_scheme = https
```

You should now have this:

```
quarkus.hibernate-search.elasticsearch.hosts = es1.mycompany.com
quarkus.hibernate-search.elasticsearch.protocol = https
quarkus.hibernate-search.elasticsearch.discovery.enabled = true
```
````

Reply:
Added, thanks.